### DISCUSSION TITLE
Database error could not save new user

### DISCUSSION THREAD

**Discussion by outstager** [supabase/supabase#10610] [Questions]
I'm getting database error - could not save new users. I need to save the id as busadminuserid rather than id.

create table busadmin(
  busadminuserid uuid,
  sal varchar(3),
  email varchar(120),
  firstname varchar(30),
  lastname varchar(30),
  username varchar(30),
  mobile varchar(15),
  
  primary key(busadminuserid)
);

ALTER TABLE public.busadmin
  ENABLE ROW LEVEL SECURITY;

CREATE POLICY
  "Can only view own profile data."
  ON public.busadmin
  FOR SELECT
  USING ( auth.uid() = busadminuserid );

CREATE FUNCTION
  public.create_profile_for_new_user()
  RETURNS TRIGGER AS
  $$
  BEGIN
    INSERT INTO public.busadmin (busadminuserid,email)
    VALUES (NEW.busadminuserid,NEW.email);
    RETURN NEW;
    END IF;
  END;
  $$ LANGUAGE plpgsql SECURITY DEFINER;

CREATE TRIGGER
  create_profile_on_signup
  AFTER INSERT ON auth.users
  FOR EACH ROW
  EXECUTE PROCEDURE
    public.create_profile_for_new_user();

**Comment by GaryAustin1** [comment]
You used NEW.busadminuserid.  That column does not exist in auth.users.  I assume you want NEW.id there. 

Also you can usually look in the Postgres log for any errors and get more info.
